net/instaweb/rewriter/render_blocking_html_computation.cc

```diff
@@ -100,8 +100,11 @@ void RenderBlockingHtmlComputation::Compute(const GoogleString& url) {
   }
 
   bool is_authorized = false;
+  // Note: this is for HTML, so may be kReconstruction would be appropriate,
+  // but it's tough to audit since this doesn't seem to have any users?
```

Comment:
Yes, remove it.  I'm not sure what it was used for but it's time to bring out our dead :)